mslib/retriever.py

```diff
@@ -246,9 +246,9 @@ def main():
             width, height = int(round(ax_bounds[2])), int(round(ax_bounds[3]))
             req = requests.get(
                 url, auth=tuple(config["WMS_login"][url]),
-                params={"version": "1.1.1", "request": "GetMap", "format": "image/png",
-                        "exceptions": "application/vnd.ogc.se_xml",
-                        "srs": "VERT:LOGP", "layers": layer, "styles": style,
+                params={"version": "1.3.0", "request": "GetMap", "format": "image/png",
+                        "exceptions": "XML",
+                        "crs": "VERT:LOGP", "layers": layer, "styles": style,
                         "dim_init_time": init_time, "time": time,
                         "width": width, "height": height,
                         "path": ",".join(f"{wp[0]:.2f},{wp[1]:.2f}" for wp in wps),
```

Comment:
I wasn't sure how to use the retriever, and ultimately unsure how to test it.
It would be very dandy if this class could somehow be integrated into the test suite, in order not to run into issues with it.
Anyway, this still needs to be tested and I don't know how, so I would love to hear your input.

Reply:
@joernu76 please try this version.

Reply:
The retriever has not really grown beyond being a prototype. If it breaks, it breaks. We have a major working package for this and I might tackle that during the next campaign if it hasn't until then.